Assets/Tests/PlayModeTests/ViewsTests.cs

```diff
@@ -786,6 +810,18 @@ public void ViewServiceSetup()
             Assert.AreEqual(0, Countly.Instance.Views._eventService._eventRepo.Count);
         }
 
+        // Set up simple view with base validation with base viewName
+        public string StartBaseViewWithValidation()
+        {
+            string viewId = _viewService.StartView(viewName);
```

Comment:
I know there will be lots of duplication but for the reviewer to easily parse a test, validation should be under a test and, creating an object and validation should not be mixed together. source: K